statsmodels/base/model.py

```diff
@@ -1311,6 +1313,129 @@ def f_test(self, r_matrix, cov_p=None, scale=1.0, invcov=None):
                              invcov=invcov, use_f=True)
         return res
 
+    @staticmethod
+    def _is_lr_test_invalid(model, attribute, expected_value):
+        """
```

Comment:
needs a one line description for the doc rendering, (doc standard even if the function doesn't go into the sphinx docs)